docs/exchanges/gate-io.md

```diff
@@ -8,16 +8,20 @@ tags:
 
 ## 📁 [Connector folder](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/gate_io)
 
+## 📝 Signup
+
+* Hummingbot referral link - <https://www.gate.io/signup/5868285>
+* Sign up with Gate.io using Hummingbot's referral link for a 10% discount!
```

Comment:
* Sign up with Gate.io using Hummingbot's referral link for a 10% discount off trading fees!